src/modules/iotjs_module_tls.c

```diff
@@ -246,23 +243,23 @@ JS_FUNCTION(TlsContext) {
   // User provided trusted certificates
   jerry_value_t jcert_auth =
       iotjs_jval_get_property(joptions, IOTJS_MAGIC_STRING_CA);
+  iotjs_string_t bjcert_auth = iotjs_string_create();
```

Comment:
For the variable maybe the `cert_auth_string` would be a better name. The `b` prefix is a bit awkward IMHO.